crypto/asn1/a_strnid.c

```diff
@@ -256,6 +256,7 @@ static ASN1_STRING_TABLE *stable_get(int nid)
         rv->mask = tmp->mask;
         rv->flags = tmp->flags | STABLE_FLAGS_MALLOC;
     } else {
+        rv->nid = NID_undef;
```

Comment:
if you replace the malloc above with zalloc, then this line (and setting rv->make) aren't needed.